examples/backends/vllm/launch/agg_multimodal_router.sh

```
@@ -1,262 +1,201 @@
-#   - No extra network hop for base64 image data
-#   - Model-agnostic (no Qwen-specific token expansion)
-#   - Single HF processor call (frontend only; backend skips via NIXL, when enabled)
+# Run inside the dynamo container that already has /workspace mounted.
 
 set -euo pipefail
-
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 DYNAMO_ROOT="$(cd "${SCRIPT_DIR}/../../../.." && pwd)"
 cd "${DYNAMO_ROOT}"
+# shellcheck source=../../../common/gpu_utils.sh
+source "${SCRIPT_DIR}/../../../common/gpu_utils.sh"
```

Shellcheck (0.11.0)

[info] Not following: ../../../common/gpu_utils.sh was not specified as input (see shellcheck -x).

(SC1091)